guides/team-and-project-scheduling

```diff
@@ -0,0 +1,67 @@
+# Team and project scheduling
+
+***If there are any questions about anything mentioned in this guide, put a message in the #scheduling slack channel - most people in this channel have a good level of knowledge of 10kft and should be able to offer clarification.***
+
+- [What we use 10kft for and why](## What we use 10kft for and why)
+ 
+## What we use 10kft for and why
+
+### Forecasting for sales
+Our sales team secure us new work, and so need to be able to use 10kft to understand team availability and capacity. They also need to know the skill shapes we have available. This helps us to match people to projects.**
```

Comment:
There are two asterisks at the end of this line that have no matching pair. I think they should be removed?